src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
> We just need to update the runtime.json file, after we've done testing, and the new RID will start to work in the fallback graph.

I agree with all of the above. Specific to this statement, when runtime.json is updated, it translates to us shipping a new SharedFX that supports the new RID and, at that time, we will be updating base fallback RID to the _updated_ latest RID.